radicle-cli/src/commands/issue.rs

```diff
@@ -359,16 +361,89 @@ pub fn run(options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
     Ok(())
 }
 
+// FIXME: this is splitting a long string into a small chunks,
+// only because our UI component is not take into count this case,
+// and we are splitting the line by `\n` and not by string length!
+//
+// We should fix the UI component, and remove the function here
+fn sub_strings(string: &str, sub_len: usize) -> Vec<&str> {
+    let mut subs = Vec::with_capacity(string.len() / sub_len);
+    let mut iter = string.chars();
+    let mut pos = 0;
+
+    while pos < string.len() {
+        let mut len = 0;
+        for ch in iter.by_ref().take(sub_len) {
+            len += ch.len_utf8();
+        }
+        subs.push(&string[pos..pos + len]);
+        pos += len;
+    }
+    subs
+}
+
 fn show_issue(issue: &issue::Issue) -> anyhow::Result<()> {
     let tags: Vec<String> = issue.tags().cloned().map(|t| t.into()).collect();
     let assignees: Vec<String> = issue.assigned().map(|a| a.to_string()).collect();
 
-    term::info!("title: {}", issue.title());
-    term::info!("state: {}", issue.state());
-    term::info!("tags: [{}]", tags.join(", "));
-    term::info!("assignees: [{}]", assignees.join(", "));
+    let mut attrs = Table::<2, Paint<String>>::new(TableOptions {
+        spacing: 2,
+        ..TableOptions::default()
+    });
+
+    attrs.push([
+        term::format::tertiary("Title".to_owned()),
+        term::format::bold(issue.title().to_owned()),
+    ]);
+    attrs.push([
+        term::format::tertiary("Stage".to_owned()),
+        term::format::bold(issue.state().to_string()),
+    ]);
+    attrs.push([
+        term::format::tertiary("Tags".to_owned()),
+        term::format::highlight(tags.join(", ")),
+    ]);
+
+    let assin = if assignees.is_empty() {
+        term::format::bold("No one".to_owned())
```

Comment:
`n/a` using `dim()`

Reply:
What is the difference with the bolt? sorry for the stupid question

Reply:
It's the opposite basically. Bold makes it brighter, Dim makes it darker.

Reply:
Oh nice! thanks